shared/src/dispatchers/ses/sendBulkTemplatedEmail.js

```diff
@@ -38,6 +38,8 @@ exports.sendBulkTemplatedEmail = async ({
         },
         ReplacementTemplateData: JSON.stringify(destination.templateData),
       })),
+      ReturnPath:
```

Comment:
Why do we require the `BOUNCED_EMAIL_RECIPIENT` for deployment if there's a default?

Reply:
Good call, I just removed the requirement.